task.sh

```diff
@@ -6,38 +6,42 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir  -p ./task/dir3/dir4
+mkdir  ./task/dir1
+mkdir  ./task/dir2
 # изменяем текущую директорию на task
-
+cd task
 # создаём пустой файл task/dir2/empty
-
+touch ./dir2/empty
 # создаём файл task/dir2/hello.sh с таким содержанием:
 # #!/bin/bash
 # echo "$1, привет!"
-
+echo '#!/bin/bash
+echo "$1, привет!"' > ./dir2/hello.sh
 # устанавливаем для task/dir2/hello.sh права rwxrw-r--
-
+chmod 764 ./dir2/hello.sh
 # сохраняем список файлов task/dir2 в task/dir2/list.txt
-
+ls ./dir2 > ./dir2/list.txt 
```

Comment:
Можно везде без `./`
```suggestion
ls dir2 > dir2/list.txt 
```